lib/apiservers/engine/backends/container.go

```diff
@@ -143,6 +152,39 @@ func NewContainerBackend() *Container {
 	}
 }
 
+// Retry retries the given function DefaultRetryCount times while sleeping some time between unsuccesfull attempts
+func Retry(op func() error) error {
+	defer trace.End(trace.Begin(""))
+
+	var err error
+
+	// DefaultRetryInterval -> [DefaultRetryInterval - delta, DefaultRetryInterval + delta]
+	delta := DefaultRetryInterval.Seconds() * DefaultRandomizationFactor
+	min := int(DefaultRetryInterval.Seconds() - delta)
+	max := int(DefaultRetryInterval.Seconds() + delta)
+
+	for retry := 1; retry <= DefaultRetryCount; retry++ {
+		err = op()
+		if err != nil && IsConflictError(err) {
+			// Pick a random integer from [min, max] range and multiply it with the index
+			next := time.Duration(retry * (rand.Intn(max-min+1) + min))
```

Comment:
May want to look at how we do backoff here: https://github.com/vmware/vic/blob/master/pkg/vsphere/tasks/waiter.go#L76. Maybe make the backoff code modular so both places can use it?